specification/README.md

```diff
@@ -1096,6 +1095,22 @@ The following example colors features with a height above 90 as red and the othe
 
 For complete details, see the [Declarative Styling](./Styling/) specification.
 
+
+## Version History
+
+- [3D Tiles 1.0](https://github.com/CesiumGS/3d-tiles/tree/1.0) _(2018-12-14)_, approved as the [3D Tiles Specification 1.0](http://docs.opengeospatial.org/cs/18-053r2/18-053r2.html) by the Open Geospatial Consortium  
+- [3D Tiles 1.1](https://github.com/CesiumGS/3d-tiles)
+  - Additions:
+    - Support for structured Metadata that can be associated with tilesets, tiles, tile content, and tile content groups
+    - Directly support glTF assets as tile contents
+    - Support for multiple tile contents
+    - Support for implicit tiling schemes
+  - Deprecations:
+    - The original tile formats (b3dm, i3dm, pnts, and cmpt) are deprecated in favor of glTF content
+    - The `tileset.properties` are deprecated, in favor of the more versatile metadata support
```

Comment:
The version history should go in the parent README to provide a better separation between the version history and the specification itself.